frontend/src/lib/hooks/api/topics.ts

```diff
@@ -50,10 +51,18 @@ export function useTopics(props: GetTopicsRequest) {
   );
 }
 export function useTopicDetails(props: GetTopicDetailsRequest) {
-  return useQuery(topicKeys.details(props), () => api.getTopicDetails(props));
+  return useQuery(
+    topicKeys.details(props),
+    () => api.getTopicDetails(props),
+    QUERY_REFETCH_ON_WINDOW_FOCUS_OFF_OPTION
```

Comment:
the hooks themselves might be used in multiple different places. I don't see a problem with refetching the defails when e.g. viewing topic details (which is okay since you can't edit anything in there), but this should have a differen behavior in topic *edit* view, can we adjust it like that?